src/Grpc/JsonTranscoding/test/Microsoft.AspNetCore.Grpc.JsonTranscoding.IntegrationTests/RouteTests.cs

```diff
@@ -105,4 +103,145 @@ Task<HelloReply> UnaryMethod(ComplextHelloRequest request, ServerCallContext con
         // Assert
         Assert.Equal("Hello complex_greeter/test2/b last_name!", result.RootElement.GetProperty("message").GetString());
     }
+
+    [Fact]
+    public async Task SimpleCatchAllParameter_PrefixSuffixSlashes_MatchUrl_SuccessResult()
+    {
+        // Arrange
+        Task<HelloReply> UnaryMethod(HelloRequest request, ServerCallContext context)
+        {
+            return Task.FromResult(new HelloReply { Message = $"Hello {request.Name}!" });
+        }
+        var method = Fixture.DynamicGrpc.AddUnaryMethod<HelloRequest, HelloReply>(
+            UnaryMethod,
+            Greeter.Descriptor.FindMethodByName("SayHelloComplexCatchAll4"));
+
+        var client = new HttpClient(Fixture.Handler) { BaseAddress = new Uri("http://localhost") };
+
+        // Act
+        var response = await client.GetAsync("/v1/greeter//name/one/two//").DefaultTimeout();
```

Comment:
Intentional?

Reply:
Yes. The purpose of this test - `SimpleCatchAllParameter_PrefixSuffixSlashes_MatchUrl_SuccessResult` - is to test extra slashes.